main.go

```diff
@@ -62,10 +65,17 @@ func main() {
 				}).Dial,
 			},
 		}
-		var queueService queue = newQueueService(&messageProducer)
+		var queueService queue
+		if *proxyAddress != "" {
```

Comment:
There was no overriding reason to remove the existing functionality completely.  The exact deployment details will depend on how we're looking to migrate the different concepts over. If we need both pipelines running, then we can just add an additional service.